src/AutorestSwift/View Models/PropertyViewModel.swift

```diff
@@ -40,31 +40,7 @@ struct PropertyViewModel {
         self.name = schema.name.toCamelCase
         self.comment = ViewModelComment(from: schema.description)
         let required = schema.required ?? false
-        self.type = getType(from: schema.schema, optional: !required)
+        self.type = schema.schema.swiftType + (!required ? "?" : "")
```

Comment:
Yes, `swiftType` should operate like `getType` did, accepting an `optional: Bool` parameter to handle this in a centralized location.